src/doc/trpl/README.md

```diff
@@ -1,21 +1,25 @@
-% The Rust Programming Language
+% The Rust Programming Language Book
```

Comment:
I see what you're going for here, but this isn't the title. "Book" is redundant.